kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -51,6 +51,9 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
 
   def submitBackgroundOperation(r: Runnable): Future[_] = execPool.submit(r)
 
+  def submitTimeoutChecker(r: Runnable, interval: Long, unit: TimeUnit): Future[_] =
+    timeoutChecker.scheduleWithFixedDelay(r, interval, interval, unit)
```

Comment:
we do need this `timeoutChecker` too, we can use `execPool` directly

Reply:
ok, I will do it.